### PR TITLE
Cleaned up Process* classes

### PR DESCRIPTION
**Pull request by marcusnaslund**
There are some possible leaks from iterators remaining, and the style isn't perfectly ours, but at least we get rid of 90 lines.
@sebastianbaginski ?

### CODE REVIEW

File: source/io/native/ProcessUnix.ooc [hunk 1]
```diff
@@ -71,40 +66,22 @@ ProcessUnix: class extends Process {
 			}
 
 			message = message + "\n"
-			if (stdErr) {
+			if (stdErr)
 				stdErr write(message)
-			} else {
+			else
 				stderr write(message)
```
Comment (sebastianbaginski) [inline source/io/native/ProcessUnix.ooc]: Could it work with the `??` operator:
```ooc
(stdErr ?? stderr) write(message)
```
?

Comment (sebastianbaginski) [inline source/io/native/ProcessUnix.ooc]: I'm just curious, you don't have to change it.

**Comment by sebastianbaginski** [comment]
Looks ok.

File: source/io/native/ProcessUnix.ooc (hunk 1 above)
Comment (marcusnaslund) [inline source/io/native/ProcessUnix.ooc]: Cool, it does! I won't change it though.

Comment (sebastianbaginski) [inline source/io/native/ProcessUnix.ooc]: Great, thanks for testing

**Pull request merged**